### DISCUSSION TITLE
钱包历史有收入4个xch，钱包为零，一周未到账

### DISCUSSION THREAD

**Discussion by heiseshupo** [Chia-Network/chia-blockchain#5178] [Support]
![未到账1](https://user-images.githubusercontent.com/81500563/118387266-263b6e00-b650-11eb-891c-cee70fe55433.png)

地址为：
xch1uwhd8qhfe07sz5mjwrajcnr6wj2m4nh2nkq682963va98l8mr2gsn7kn93

xch1uwhd8qhfe07sz5mjwrajcnr6wj2m4nh2nkq682963va98l8mr2gsn7kn93

xch1cw6dl9mwhunf0cf3hlkl83wqkr40dg9dmwr53mkfduljzyh9tk3qhzpyy6

xch1xxynl05fs35y6aqaml4pa854pfzl9vv28kqrrx3jhx9v7897jnqsh5qy34

第三个第四个地址更奇特，居然还是2020年的块，杂质是啥意思。还有在区块链浏览器上公钥无法搜索到数据。

请大神，指导一下。我这个钱包至今币没到账，是1.15版本点新地址的bug？还是网络没同步（本地显示同步，8444也开了）？

还是被盗走了？本人未加入任何矿池，一直用官方软件p图，挖矿，没有用第三方软件。唯一就公网ip映射不会操作，让人远程了一次。

![DCBCBF56409D89C33B7D613D812CBC76](https://user-images.githubusercontent.com/81500563/118387298-5edb4780-b650-11eb-97c8-69231442898d.jpg)

**Comment by Heuer212** [comment]
我也这样，不知道咋回事

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
